nix/dapp.nix

```diff
@@ -0,0 +1,740 @@
+# This file has been generated by dapp2nix 0.0.2, do not edit.
+{ solidityPackage, solc, dapp2 }:
+
+let
+  inherit (builtins) map listToAttrs attrNames attrValues;
+  mapAttrs = if (builtins ? mapAttrs)
+    then builtins.mapAttrs
+    else f: attrs:
+      listToAttrs (map
+        (name: { inherit name; value = f name attrs."${name}"; })
+        (attrNames attrs));
+
+  defaults = {
+    inherit solc;
+    test-hevm = dapp2.test-hevm;
+    doCheck = true;
+  };
+
+  package = spec: let
+    spec' = defaults // (removeAttrs spec [ "repo'" "src'" ]);
+  in solidityPackage (spec' // {
+    deps = map (spec:
```

Comment:
Maybe it would be more clear if `deps` was defined inside the `let` block?

Reply:
Yes, absolutely! Have updated https://github.com/icetan/dapp2nix